torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -9708,6 +9709,14 @@ TEST_F(NVFuserTest, FusionPersistentNormLocalShared_CUDA) {
   const float kEps = 1e-5;
   auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
 
+  auto properties = at::cuda::getDeviceProperties(0);
+  const size_t required_smem_size =
+      (dimy - static_size) * sizeof(float) + TIDX * sizeof(float);
```

Comment:
Not necessary in this PR, but it would be good to use the inferred shared memory size from `FusionExecutor::computeSharedMemory`, instead of manually calculating it here.

Reply:
I think that's part of the check actually. We want to make sure that the aliasing is accurately analyzed and only this size is required. If aliasing analysis fails somehow, `FusionExecutor::computeSharedMemory` could just return a larger size, and the test would be just skipped, but in that case we would want the test to fail. So in this case checking with this `required_smem_size` is more appropriate.

Reply:
I see, makes sense